### PR TITLE
[TTAHUB-3484] Insert standard goals 

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change
A database migration to insert the new standard goals into the database

## How to test
- Confirm that the language matches the PDF provided by OHS
- Confirm that the goals are selectable in the AR & RTR

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3484

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Before merge to main

- [ ] OHS demo complete
- [ ] Ready to create production PR

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: src/migrations/20241024002207-insert-standard-goals.js
Comment (GarrettEHill) [inline src/migrations/20241024002207-insert-standard-goals.js:1]: Did you get a new updated list? Newer then last week, as they said there are a few changes.

Comment (thewatermethod) [reply src/migrations/20241024002207-insert-standard-goals.js:1]: I did not, I will post in the customer channel and see what's up

**Comment by thewatermethod** [comment]
Language updated with most recent 10/24

File: src/migrations/20241024002207-insert-standard-goals.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    },
+  ),
+
+  down: async (queryInterface) => queryInterface.sequelize.transaction(
+    async (transaction) => {
+      await prepMigration(queryInterface, transaction, __filename);
+
+      await queryInterface.sequelize.query(`
+        DELETE FROM "GoalTemplates"
+        WHERE "creationMethod" = 'Curated' AND "templateName" IN (${standardGoalTexts.map((templateName) => `'${templateName}'`).join(',')});
```
Comment (GarrettEHill) [inline src/migrations/20241024002207-insert-standard-goals.js:52]:
```suggestion
        WHERE "creationMethod" = '${CREATION_METHOD.CURATED}' AND "templateName" IN (${standardGoalTexts.map((templateName) => `'${templateName}'`).join(',')});
```

**Review by AdamAdHocTeam — COMMENTED** [review]
Diff on first goal:

image

**Comment by thewatermethod** [comment]
> Diff on first goal:
> 
> image

Discussed with Adam, seems to be a quirk with the document he had, hyphen is correct

**Review by nvms — APPROVED** [review]

File: src/migrations/20241024002207-insert-standard-goals.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  '(Program Structure) The recipient will implement management and program structures that provide eﬀective oversight and administration and meet the needs of the staﬀ, families, and communities.',
+  '(Disaster Recovery) The recipient will implement systems and services to support children, families, and staﬀ with recovering from disasters.',
+  '(RAN investigation) The recipient will implement systems and services to address a reported child incident during the RO investigation.',
+];
+const standardGoal = (templateName) => ({
+  creationMethod: CREATION_METHOD.CURATED,
+  hash: md5(templateName),
+  createdAt: new Date(),
+  updatedAt: new Date(),
+  templateNameModifiedAt: new Date(),
```
[the remark this answers is not in the document]
Comment (thewatermethod) [reply src/migrations/20241024002207-insert-standard-goals.js]: 26494f1

**Review by GarrettEHill — APPROVED** [review]

**Pull request merged**